website_sale_stock_provisioning_date/static/src/js/website_sale_stock_provisioning_date.js

```diff
@@ -0,0 +1,15 @@
+/* Copyright 2020 Tecnativa - Ernesto Tejeda
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+odoo.define("website_sale_stock_provisioning_date.load", function (require) {
+    "use strict";
+    var ajax = require("web.ajax");
+    var core = require("web.core");
+    var QWeb = core.qweb;
+    var load_xml = ajax.loadXML(
+        "/website_sale_stock_provisioning_date/static/src/xml/website_sale_stock_product_availability.xml",
```

Comment:
Is this patching mode still needed or could we achive it with qweb assets?

Reply:
@CarlosRoca13 can you answer?

Reply:
I'm not sure, the original view is not owl="1"... I will test it

Reply:
We can't extend it using qweb assets because the base view is not charged at this assets